app/models/services/facebook.rb

```diff
@@ -23,16 +23,16 @@ def post_to_facebook(url, body)
   end
 
   def create_post_params(post)
-    message = post.message.plain_text_without_markdown
-    if post.photos.any?
-      message += " " + short_post_url(post, protocol: AppConfig.pod_uri.scheme,
-                                            host: AppConfig.pod_uri.authority)
+    post_message = post.message.plain_text_without_markdown
+
+    if post_message.empty? && post.photos.none?
+      message = post_message
+    else
+      post_url = short_post_url(post, protocol: AppConfig.pod_uri.scheme, host: AppConfig.pod_uri.authority)
+      message = "#{post_message} (via #{post_url} )"
     end
 
-    {message: message,
-     access_token: access_token,
-     link:  post.message.urls.first
-    }
+    {message: message, access_token: access_token, link: post.message.urls.first}
```

Comment:
> Align the elements of a hash literal if they span more than one line

It means to write it like this:

```ruby
    {
      message:      message,
      access_token: access_token,
      link:         post.message.urls.first
    }
```

But I think it's OK to have it in one line here.

Reply:
I tried that and when I ran style check it still complained.  I'll try it again, I much prefer it over the one line...